src/components/CrudContainer/CrudContainer.tsx

```diff
@@ -1,17 +1,32 @@
 import { useState } from "react"
 import { FormModal } from '../Modal/FormModal'
 import Button from '@mui/material/Button'
+import { createWine } from '../../api/wine_api'
+import { Wine, WineData } from "../../interfaces/wine_interface"
 
-export const CrudContainer = () => {
-    const [openAddModal, setOpenAddModal] = useState(false)
+export const CrudContainer: React.FC<{
+  wines: Wine[]
+  setWines: (wines: Wine[]) => void
+}> = ({wines, setWines}) => {
+  const [openAddModal, setOpenAddModal] = useState(false)
```

Comment:
nitpick: `openAddModal` sounds to me like a function. For state I would prefer using `is` like `isAddModalOpen`.